src/App.js

```diff
@@ -1,26 +1,78 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
+import { request } from './api/api';
 
-export class App extends Component {
-  state = {
-    movies: data,
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+  const [query, setQuery] = useState('');
+  const [film, findFilm] = useState(null);
+  const [error, findError] = useState(false);
```

Comment:
```suggestion
  const [film, setFilm] = useState(null);
  const [error, setError] = useState(false);
```
The names of this functions should start with `set`. You are actually 'setting' new value to the state